draft-ietf-quic-recovery.md

```diff
@@ -1433,7 +1437,7 @@ UpdateRtt(ack_delay):
   // min_rtt ignores acknowledgment delay.
   min_rtt = min(min_rtt, latest_rtt)
   // Limit ack_delay by max_ack_delay after handshake
-  // confirmation. Note that ack_delay is 0 for
+  // confirmation. Note that ack_delay is clamped to 0 for
```

Comment:
ack_delay is an input to this function, and I didn't remember any external constraint that the provided function argument would be zero in this case.